### PR TITLE
API Release

### PR DESCRIPTION
**Pull request by albertcui**

### CODE REVIEW

File: _posts/2018-04-17-changes-to-the-api.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  Important Changes to the OpenDota API
+date: 2018-04-17
+summary: Introducing the Free and Premium Tiers
+---
+**IMPORTANT: Users making more than 50,000 API calls per month will need to sign up for an API key in order to keep using the API.**
```
Comment (howardchung) [inline _posts/2018-04-17-changes-to-the-api.md]: "in order to enable usage beyond this limit"?

File: _posts/2018-04-17-changes-to-the-api.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+As we’re committed to both providing the service free to our users and avoiding ad networks which monetize through your
+data and browsing habits, we’ve decided to begin offering increased rate and call limits for registered high-volume users
+on a premium plan. This will have no effect on the average end user of the site and app while ensuring heavy API users help
+offset our costs.
+
+We’re committed to continuing to support hobbyists and community developers with a Free Tier of 50,000 API calls per month.
+Based on our survey of our API users, we feel the Free Tier limits will appropriately satisfy most hobbyist use cases.
+
+**IMPORTANT: Users making more than 50,000 API calls per month will need to sign up for an API key in order to keep using the API.**
```
Comment (howardchung) [inline _posts/2018-04-17-changes-to-the-api.md]: duplicated?

Comment (albertcui) [reply _posts/2018-04-17-changes-to-the-api.md]: intentional

File: _posts/2018-04-17-changes-to-the-api.md (hunk 1 above)
Comment (albertcui) [reply _posts/2018-04-17-changes-to-the-api.md]: changed

**Review by howardchung — APPROVED** [review]

**Pull request merged**